packages/firestore/src/platform_browser/browser_platform.ts

```diff
@@ -72,4 +74,22 @@ export class BrowserPlatform implements Platform {
   btoa(raw: string): string {
     return btoa(raw);
   }
+
+  randomBytes(nBytes: number): Uint8Array {
+    validatePositiveNumber('randomBytes', 1, nBytes);
```

Comment:
This should be a `debugAssert` since we are not validating a public API call.

You should also allow calls with `nBytes: 0` (unless you have to write special code for this).

Reply:
Done.